src/www/ui/api/Models/UploadSummary.php

```diff
@@ -119,6 +125,7 @@ public function getArray()
     return [
       "id"                      => $this->uploadId,
       "uploadName"              => $this->uploadName,
+      "assigneeId"              => $this->assigneeId,
```

Comment:
IMO, calling it `assignee` will be much better.

Reply:
Edited